doc/en/user/source/installation/osx_jaierror.txt

```diff
@@ -0,0 +1,15 @@
+.. warning:: If you encounter the following error during startup, you may have some invalid JAI jars from the default Mac Java install:
+
+    .. code-block:: bash
+
+        java.lang.NoClassDefFoundError: Could not initialize class javax.media.jai.JAI
+    
+    To fix this error, locate your Java installation folder, and delete the following jars:

```

Comment:
These can also be found in the ~/Library/Java/Extensions folder (and are automatically installed in this location by uDig).

![jai-extensions](https://user-images.githubusercontent.com/629681/29643704-a10b7d3a-8825-11e7-9ed6-aae86057cf18.png)